settings.gradle.kts

```diff
@@ -129,12 +129,12 @@ dependencyResolutionManagement {
       val bouncycastleVersion = "1.70"
       val daggerVersion = "2.42"
       val eclipseCollectionsVersion = "10.4.0"
-      val grpcVersion = "1.48.2" // higher 'grpc-netty' versions lead to hanging integration tests
+      val grpcVersion = "1.45.1" // higher 'grpc-netty' versions lead to hanging integration tests
```

Comment:
Is it reasonable to add a comment here pointing out the (extremely) tight interdependence of `grpcVersion` and `nettyVersion`?  It should be _somewhere_ to remind the next person who changes one or the other that mismatches here lead to problems.  And this seems to be _the_ place in the code that deals with versions ....

(Comment could just be a URL pointing ... somewhere ... a closed issue? A "discussion" topic that we could create to "discuss" this?)

Reply:
We removed this by switching to only using the helidon repackage (separate PR).
That reduces to a single dependency that we should consider "sensitive to change".